admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -0,0 +1,57 @@
+<div
+  class="
+    <%= stimulus_id %>
+    rounded-lg
+    border
+    border-gray-100
+    border-gray-100
+    overflow-hidden
+  "
+  data-controller="<%= stimulus_id %>"
+>
+  <table class=" table-auto w-full border-collapse ">
+    <colgroup>
+      <% @columns.each do |column| %>
+        <col class="<%= column.class_name %>">
+      <% end %>
+    </colgroup>
+
+    <thead class="bg-gray-15 color-gray-100">
+      <tr>
+        <% @columns.each do |column| %>
+          <th
+            class="
+              border-b
+              border-gray-100
+              py-[12px]
+              px-[16px]
+              text-[#4f4f4f]
+              text-left
+              text-[14px]
+              font-[600]
+              line-[120%]
+            "
+          ><%= column.name %></th>
+        <% end %>
+      </tr>
+    </thead>
+
+    <tbody class="bg-white text-[14px] line-[150%] text-black">
```

Comment:
I think that if some of us are using it and some don't we'll end up having a lot of troubles when the file is changed later, because it's going to reformat the whole file, even on lines unrelated to the current changes. My vote is to disable it or enable it for anyone (no strong opinion on them, but I don't know the implications of having it enabled on a open source project)